go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openshift-kni/cluster-group-upgrades-operator
 
-go 1.23.0
+go 1.24.0
```

Comment:
Is dependabot supposed to bump golang version like this? This would require updating the builder images too

@jc-rh @sudomakeinstall2

Reply:
no,  we can't take this.